packages/gamut-labs/src/ListSection/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ReactElement, useState } from 'react';
+
+import { PageSection } from '..';
+
+export type ListSectionProps<T> = {
+  title: string;
+  /**
+   * Array of elements.
+   */
+  listItems: T[];
+  /**
+   * Render function that will render each element in the list items.
+   */
+  renderItem: (item: T, index?: number) => ReactElement;
```

Comment:
The `?` means this component _might_ provide an `index: number` when `renderItem` is called. I see in the code below that it's never provided, so you probably want this:

```suggestion
  renderItem: (item: T) => ReactElement;
```